### PR TITLE
PAE 660 - Move login style to main page

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
The login and registration templates have been removed, the previous behavior of the login page was moved to the main page.

**Theme:** pearson-vue-theme

**Ticket:** [PAE-660](https://pearsonadvance.atlassian.net/browse/PAE-660)

**Screenshots:**

**Desktop:**

image

**Mobile:**

image

image

### CODE REVIEW

File: edx-platform/pearson-vue-theme/lms/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
+  </div>
+
+  <div class="signin-container">
+
+    <div class="toggle-form">
+      <h1>${_("Welcome!")}</h1>
+      <br>
+      <span class="text">${_("Sign in or create an account to access your courseware.")}</span>
+
```
Comment (ivanvgh) [inline edx-platform/pearson-vue-theme/lms/templates/index.html]: @luismorenolopera is this blank line necessary? same with line 12, 17 and 19.

Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/index.html]: @ivanvgh I think we can leave the space of line 12, the rest I have removed

**Review by ivanvgh — APPROVED** [review]

File: edx-platform/pearson-vue-theme/lms/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  </div>
+</div>
+
+% if show_homepage_promo_video:
+  <section id="video-modal" class="modal home-page-video-modal video-modal">
+    <div class="inner-wrapper">
+      <iframe title="YouTube Video" width="640" height="360" src="//www.youtube.com/embed/${homepage_promo_video_youtube_id}?showinfo=0" frameborder="0" allowfullscreen></iframe>
+    </div>
+  </section>
+% endif
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/index.html:40]: Is this block being used? @luismorenolopera

Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/index.html]: Is this block being used? @luismorenolopera

Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/index.html]: no, this is something that comes in the original template.  Should I remove everything that is not used from the original template?

Comment (Squirrel18) [reply edx-platform/pearson-vue-theme/lms/templates/index.html]: If it's not being used, I think it's fine to remove it. @luismorenolopera

Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/index.html]: I have removed it along with other lines that are not in use @Squirrel18

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**